### PR TITLE
Tmz 1.6.0 quadratic equation

### PR DESCRIPTION
**Pull request by t-m-z**
solves quadratic equations

### CODE REVIEW

File: crowdin.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+project-id=445424
```
Comment (Mike-3) [inline crowdin.properties:1]: please remove this file.
In PR Tmz 1.6.0 maya calendar in .gitignore

File: lib/widgets/tools/science_and_technology/quadratic_equation.dart [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
```
Comment (Mike-3) [inline lib/widgets/tools/science_and_technology/quadratic_equation.dart:3]: remove unused imports

**Comment by Mike-3** [comment]
character input posibile. Please change
Possibly, for example, 1/3 to be accepted as a valid input (use math_expressions) 
Possibly a preview for the equation that will be solved or a static example equation where you can see where a, b and c are used.

File: crowdin.properties (hunk 1 above)
Comment (t-m-z) [reply crowdin.properties]: done

**Comment by t-m-z** [comment]
> character input posibile. Please change
> Possibly, for example, 1/3 to be accepted as a valid input (use math_expressions)
> Possibly a preview for the equation that will be solved or a static example equation where you can see where a, b and c are used.

preview with a static example is done, but I have no idea how to solve this math expression issue. Maybe we should keep this as an enhancement

**Review by Mike-3 — APPROVED** [review]

File: lib/widgets/tools/science_and_technology/quadratic_equation.dart [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        GCWOutputText(
+          text: i18n(context, 'quadratic_equation_explanation')+'\n',
+          suppressCopyButton: true,
+        ),
+        Row(
+          children: <Widget>[
+            Expanded(
```
Comment (S-Man42) [inline lib/widgets/tools/science_and_technology/quadratic_equation.dart:51]: Currently there is no space between the Textfields. Please add some space to fit the common layout :)

You can do this by either wrap the textfields into a Container:

```
Row(
 children: [
     Container(
        child: Textfield1,
       padding: EdgeInsets.only(right: DEFAULT_MARGIN)
      ),
     Container(
        child: Textfield2,
         padding: EdgeInsets.only(left: DEFAULT_MARGIN, right: DEFAULT_MARGIN)
      ),
    Container(
        child: Textfield3,
       padding: EdgeInsets.only(left: DEFAULT_MARGIN)
      ),
]
)
```

or by adding an empty container inbetween (i am not quite sure about this, but it should work):

```
Textfield1,
Container(width: DOUBLE_DEFAULT_MARGIN),
Textfield2,
....
```

**Comment by S-Man42** [comment]
Just a minor layout issue. Ready to merge afterwards.

File: lib/widgets/tools/science_and_technology/quadratic_equation.dart (hunk 3 above)
Comment (t-m-z) [reply lib/widgets/tools/science_and_technology/quadratic_equation.dart:51]: done

**Comment by t-m-z** [comment]
PR done

**Comment by S-Man42** [comment]
We have conflicts here now :( Sorry, but this happens when merging many PRs in a row :(

**Comment by t-m-z** [comment]
solved

**Pull request merged**